docker-compose.yaml

```diff
@@ -26,6 +26,8 @@ services:
       - "--import-realm"
       - "--hostname-url=http://localhost:8081"
       - "--health-enabled=true"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Might be nice to add a code comment, at least once, just to say that this is for Linux compatibility. Because we will forget in a year's time. ;-)